protocol/requests/save_new_waku_node.go

```diff
@@ -0,0 +1,25 @@
+package requests
+
+import (
+	"strings"
+
+	"github.com/multiformats/go-multiaddr"
+)
+
+type SaveNewWakuNode struct {
+	NodeAddress string `json:"nodeAddress"`
+}
+
+func (r *SaveNewWakuNode) Validate() error {
+	if strings.HasPrefix(r.NodeAddress, "enrtree://") {
+		return nil
```

Comment:
I don,t know what the API expects, but should it have a prefix or not? If it does, it returns, but no error, so we skip the second check.
I think something is wrong.

Reply:
I found two places that check the Waku node address:

In AddWakuNodeModal.qml I noticed the following validation:
```
                StatusRegularExpressionValidator {
                    errorMessage: qsTr("Value should start with '/' or 'enr:'")
                    regularExpression: /(\/|enr:).+/
                }]
```
and in waku.go
```
	for _, addrString := range w.cfg.WakuNodes {
		if strings.HasPrefix(addrString, "enrtree://") {
                    ...
		} else {
			// It is a normal multiaddress
			addr, err := multiaddr.NewMultiaddr(addrString)
```

Reply:
Ah ok I get it now. It's just very unclear in the code. It looks like it might be a mistake. Maybe you could add a comment?